app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontCreateCustomerAccountWithAddressInformationActionGroup.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontCreateCustomerAccountWithAddressInformationActionGroup" extends="SignUpNewUserFromStorefrontActionGroup">
+        <annotations>
+            <description>EXTENDS: SignUpNewUserFromStorefrontActionGroup. Fills address fields.</description>
+        </annotations>
+        <arguments>
+            <argument name="address" type="entity"/>
+        </arguments>
+        <fillField selector="{{StorefrontCustomerAddressFormSection.company}}" userInput="{{address.company}}" after="fillGender" stepKey="fillCompany"/>
```

Comment:
@chiranjeevi-cj, looks like failures caused by you mention `after="fillGender"` but `stepKey="fillGender" `doesn't exist in actionGroup which you extend.
Could you please take a look?

Reply:
@engcom-Charlie I have changed that